applications/app/services/NewspaperQuery.scala

```diff
@@ -135,7 +135,7 @@ class NewspaperQuery(contentApiClient: ContentApiClient) extends Dates with Logg
 
   private def getNewspaperPageNumber(content: ApiContent) = content.fields.flatMap(_.newspaperPageNumber)
 
-  def lowercaseDisplayName(s: String): String = if(s.equals("UK news") || s.equals("US news")) s else s.toLowerCase()
+//  def lowercaseDisplayName(s: String): String = if(s.equals("UK news") || s.equals("US news")) s else s.toLowerCase()
```

Comment:
should we just remove this?

Reply:
Haha absolutely! For some reason I thought I had deleted it, not commented out. Lesson: Don't work late.

Reply:
that's an important lesson :)